src/cdk-experimental/menu/menu.ts

```diff
@@ -318,6 +329,16 @@ export class CdkMenu extends CdkMenuGroup implements Menu, AfterContentInit, OnI
     return this.orientation === 'horizontal';
   }
 
+  /**
+   * Return true if this menu is an inline menu. That is, it exists in isolation apart from a menu
+   * bar.
```

Comment:
My understanding is that it's more that an inline menu doesn't exist in a pop-up rather than being strictly related to a menubar.